gnome-initial-setup/pages/language/cc-language-chooser.c

```diff
@@ -661,7 +662,7 @@ cc_language_chooser_init (CcLanguageChooser *chooser)
 {
         CcLanguageChooserPrivate *priv = cc_language_chooser_get_instance_private (chooser);
         gtk_widget_init_template (GTK_WIDGET (chooser));
-        priv->showing_extra = TRUE;
+//        priv->showing_extra = TRUE;
```

Comment:
And I would just delete this line instead of commenting it out... we have git to track revision history. ;)